### DISCUSSION TITLE
Some missing keys for English Translation

### DISCUSSION THREAD

**Discussion by magnolima** [Dymerz/StarCitizen-Localization#187] [General]
I noticed that the English translation is missing some keys (actually 799 keys), any information on why? Below only few:

DataHeist_AccessCard,P
DataHeist_ECT_Access,P
DataHeist_ECT_Access_AccessCardInvalid,P
DataHeist_ECT_Access_CardConfiscated,P
DataHeist_ECT_Access_Denied,P
DataHeist_ECT_Access_Granted,P
DataHeist_ECT_Access_InsertAccessCard,P
DataHeist_ECT_Access_ReadingCard,P
DataHeist_ECT_Access_SecurityBreach,P
DataHeist_ECT_Access_SystemLockdown,P
DataHeist_ECT_Access_TerminalLocked,P
DataHeist_ECT_Access_Welcome,P
DataHeist_ECT_AccessCardInvalid,P
DataHeist_ECT_AttentionRequired_TopBarInfo,P
...
vehicle_NameVNCL_Scythe_SWARM_AI_SCAVENGER
vehicle_NameXIAN_Nox_Kue_Kue
vehicle_NameXIAN_SantokYai
vehicle_NameXIAN_Scout_Hangar

My translation (Portuguese) has all of the 799 keys.

**Comment by ROBdk97** [comment]
They have not gone missing. Instead, they have been altered by CIG. For instance, the ",P" part was either added or removed, or other minimal modifications were made. The English file remains exactly as it was extracted from the current build

**Reply by ROBdk97** [reply]
I just checked the file again. The current file in the repo is the same as from the current live build.
So you or somebody else needs to alter or add the keys to match the English files keys as it is the original file from CIG.

**Reply by magnolima** [reply]
Thank you, yes, I am using the English as base and those keys had vanished during the PTU to LIVE. For now I simply removed the tags and kept just the English base version.
